utils/src/main/scala/za/co/absa/enceladus/utils/implicits/StructFieldImplicits.scala

```diff
@@ -24,8 +24,8 @@ object StructFieldImplicits {
       Try(structField.metadata.getString(key)).toOption
     }
 
-    def getMetadataBoolean(key: String): Option[Boolean] = {
-      Try(structField.metadata.getBoolean(key)).toOption
+    def getMetadataStringAsBoolean(key: String): Option[Boolean] = {
```

Comment:
Renaming the method seems to leak too much implementation details. I'd keep it as it was.

Reply:
Would `getMetadataBooleanString` serve better? I want to distinguish from reading an actual Boolean metadata value.

Reply:
No, the intent is clearer with `getMetadataStringAsBoolean`, let's keep it that way.